requirements/default.txt

```diff
@@ -196,3 +196,6 @@ elasticsearch-dsl==0.0.11 \
 semver==2.10.2 \
     --hash=sha256:21e80ca738975ed513cba859db0a0d2faca2380aef1962f48272ebf9a8a44bd4 \
     --hash=sha256:c0a4a9d1e45557297a722ee9bac3de2ec2ea79016b6ffcaca609b0bc62cf4276
+enforce-typing==1.0.0.post1 \
+    --hash=sha256:90347a61d08e7f7578d9714b4f0fd8abd9b6bc48c8ac8d46d7f290d413afabb7 \
+    --hash=sha256:d3184dfdbfd7f9520c884986561751a6106c57cdd65d730470645d2d40c47e18
```

Comment:
We use this to enforce types data for dataclass instantiations. It's not a great library. It doesn't seem to get much maintenance. There don't seem to be better alternatives a this time.

This is good enough until we need to do nested data structures. At that point, this fails and we might want to switch from Python data classes to pydantic thing.